### PR TITLE
feat: add keys to all for loops

### LINKED ISSUE #168: Use vuejs/eslint-plugin-vue

### PR DESCRIPTION
**Pull request by rayrutjes**
By using [Vetur](https://github.com/vuejs/vetur), this was very easy to fix.

I think it's safe to close #168 for now.

### CODE REVIEW

**Comment by rayrutjes** [comment]
This needs to be tested a bit before merging.

**Comment by Haroenv** [comment]
Weird that key didn't work in #166

**Comment by rayrutjes** [comment]
Yes, that's why I wanted to try it out first and I'm now confident it works.
Must have been a typo or something, or something related to a specific version of Vue.

File: src/components/Rating.vue [hunk 1]
```diff
@@ -2,20 +2,20 @@
   <div :class="bem()" v-if="show">
     <slot name="header"></slot>
 
-    <a href="#" @click.default="clear" :class="bem('clear')" v-if="currentValue">
+    <a href="#" @click.prevent="clear" :class="bem('clear')" v-if="currentValue">
```
Comment (Haroenv) [inline src/components/Rating.vue]: This seems like a different change, right?

Comment (rayrutjes) [inline src/components/Rating.vue]: yes, but was lazy enough to not separate the commit :/
default is not a valid "modifier", "default" is.

Comment (rayrutjes) [inline src/components/Rating.vue]: Actually, I will double check this

Comment (rayrutjes) [inline src/components/Rating.vue]: Seems correct.

**Comment by julienbourdeau** [comment]
🎉 🎈

**Pull request merged**